definitions/npm/@testing-library/react_v9.x.x/flow_v0.104.x-/react_v9.x.x.js

```diff
@@ -115,189 +115,204 @@ declare module '@testing-library/react' {
     eventProperties?: TInit
   ) => boolean;
 
-  declare type Queries = { ... };
-
-  declare type RenderResult<Q: Queries = GetsAndQueries> = {|
+  declare type RenderResult<Queries = GetsAndQueries> = {|
+    container: HTMLDivElement,
+    unmount: () => void,
+    baseElement: HTMLElement,
+    asFragment: () => DocumentFragment,
+    debug: (baseElement?: HTMLElement) => void,
+    rerender: (ui: React$Element<*>) => void,
```

Comment:
`*` is depreacted, `any` should likely be sufficient here?

Reply:
Ah I didn't touch the other types besides `render` here, only moved them. I can do that in another PR in the future

Reply:
Sure!